assignments/this.js

```diff
@@ -1,26 +1,58 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
```

Comment:
good job with these questions. Your answer and short and to the point which is exactly what I like to see. well done.